### PR TITLE
Add Rythm template engine 

### PR DESCRIPTION
**Pull request by danjee**

### CODE REVIEW

File: pippo-rythm/pom.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>ro.pippo</groupId>
+            <artifactId>pippo-core</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <!-- Freemarker -->
```
Comment (decebals) [inline pippo-rythm/pom.xml]: change Freemarker with Rythm

**Comment by decebals** [comment]
Nice, Thanks.

Can you see if `Rythm` has support for `methods/extensions/handlers` (see my [comment](https://github.com/decebals/pippo/issues/144#issuecomment-112451241) from pippo-velocity implementation) because will be nice to wrote `@webjarsAt()` instead of `@pippo.webjarsAt()`.

**Comment by danjee** [comment]
Yes, this can be done, I will send a PR when changes are completed

**Comment by danjee** [comment]
I will reopen a new PR when modifications are done

**Pull request closed without merging**